remove filter and put closed opportunities in a table below the open …

RachelMuston opened the pull request:
Ok I've gotten this most of the way there! @smellems hoping you can have a look at this to see what is causing the error message and some of the filter to still show up. Also why the 'no data is available in the table' shows up.

Linked issues #91 and #97

smellems commented:
The error message is only because both tables have the same ID.
The filters would apply only to the one of the tables (Open opportunities).
The filter that allows searching in the table is added by DataTables, maybe we can remove it

The closed opportunities could be displayed not in a table.
Maybe a list in a details/summary (click to see closed opportunities)

smellems commented:
`No data is available in the table` is the Datatables message.  Already works in French

smellems commented:
I pushed a change.  See if you like it

RachelMuston commented:
I pushed a few changes: removed the filter, removed the 'showing x of n entries', put cards back under 'Closed'.
Its looking close to how I want it :)
but:
1. the 'No data is available in the table' isn't ideal - I'd rather this message be more customized (e.g. 'No open opportunities' for the Open data table and 'No closed opportunities' for the Closed data table.
2. the box under 'Open' that is going through the text of 'No data is available...'.  Want to remove this box. is it the border around the data table?

RachelMuston commented:
oooooh or to make it easier, what if the message was just 'No opportunities'. That way the message would work regardless of which data table (the open or the closed one).  And since these are the only two data tables on our site, that could work?  Where is that data tables message stored?  Is it hard coded into the API?

RachelMuston commented:
There is a small bug with the box around the words 'No open/closed opportunities'.  The box cuts through the words with Chrome browser (but not Firefox). Hoping to remove the box. But lets merge this change for now and open a new issue for the box bug.
